.github/workflows/e2e-test-kind.yaml

```diff
@@ -44,6 +44,21 @@ jobs:
         run: |
           IMAGE=velero VERSION=pr-test make container
           docker save velero:pr-test -o ./velero.tar
+  # validate boringcrypto
+  # make local uses build.sh that verifies boringcrypto
+  build-boring:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the code
+        uses: actions/checkout@v4
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+      - name: Build Velero CLI
+        if: steps.cli-cache.outputs.cache-hit != 'true'
+        run: |
+          GOEXPERIMENT=boringcrypto make local
```

Comment:
Verified via this job.

https://github.com/vmware-tanzu/velero/actions/runs/11896420547/job/33148282324?pr=8412#step:4:17
```shell
GOOS=linux \
GOARCH=amd64 \
GOBIN=$(pwd)/.go/bin \
GOEXPERIMENT=boringcrypto \
VERSION=main \
REGISTRY=velero \
PKG=github.com/vmware-tanzu/velero \
BIN=velero \
GIT_SHA=99361e9899ab61d[5](https://github.com/vmware-tanzu/velero/actions/runs/11896420547/job/33148282324?pr=8412#step:4:6)4543eef326598c1e1e347941 \
GIT_TREE_STATE=clean \
OUTPUT_DIR=$(pwd)/_output/bin/linux/amd[6](https://github.com/vmware-tanzu/velero/actions/runs/11896420547/job/33148282324?pr=8412#step:4:7)4 \
./hack/build.sh
FIPS verified for /home/runner/work/velero/velero/_output/bin/linux/amd64/velero
```